pkg/diagnostics/v2/client.go

```diff
@@ -76,6 +79,36 @@ func (c *Client) Download(id string, dst io.Writer) error {
 	}
 }
 
+// Create creates a new cluster bundle and returns its ID.
+func (c *Client) Create() (string, error) {
+	newID := uuid.NewV4()
+	req, err := c.http.NewRequest("PUT", fmt.Sprintf("%s/%s", baseURL, newID.String()), nil)
```

Comment:
```suggestion
	req, err := c.http.NewRequest("PUT", fmt.Sprintf("%s/%s", baseURL, uuid.NewV4()), nil)
```

Just curious, why is the client expected to generate the ID, instead of having a POST endpoint which would create the id and return it?

Reply:
Great question. It's a design decision I'm not particularly happy with but haven't been involved too much. Probably @janisz might be able to elaborate on it.